device_backends/LogicalNameMapping/src/LNMMathPlugin.cc

```diff
@@ -142,6 +147,21 @@ namespace ChimeraTK::LNMBackend {
 
   /********************************************************************************************************************/
 
+  void MathPluginFormulaHelper::clearParameterQueues() {
+    for(auto& p : params) {
+      bool rv = true;
```

Comment:
What is rv? Although this is a very local variable I cannot guess the name. Is it "there has been data on the queue"?